openapi-v1.yaml

```diff
@@ -6718,6 +6768,44 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /invitations/{namespace}/{group}/share/group:
```

Comment:
This should be `/invitations/{namespace}/{group}/share`

Reply:
If omit the /share then we have same path with /invitations/{namespace}/{array}/share

Reply:
Let's reuse `/invitations/{namespace}/{array}/share`, updating param list with InvitationGroupShareEmail, REST should be able to distinguish if it is an array or a group and act accordingly